lib/ecl/tests/ecl_nnc_export.cpp

```diff
@@ -74,16 +74,18 @@ int count_kw_data( const ecl_file_type * file , ecl_grid_type * grid , const cha
 
 void test_count(const char * name) {
   char * grid_file_name = ecl_util_alloc_filename(NULL , name , ECL_EGRID_FILE , false  , -1);
+  char * init_file_name = ecl_util_alloc_filename(NULL , name , ECL_INIT_FILE , false  , -1);
   ecl_grid_type * grid = ecl_grid_alloc( grid_file_name );
   ecl_file_type * grid_file = ecl_file_open( grid_file_name , 0 );
+  ecl_file_type * init_file = ecl_file_open( init_file_name , 0);
```

Comment:
The internal equionor test fails because the `ecl_file_open(init_file_name)` returns `NULL` - because there is no such file. I have updated the testdata internally, so it should work again quite soon.

Reply:
great, thanks!

Reply:
OK - then we merge!